Lookup Templates: rewrite (#1146)
DevExpress/devextreme-demos: JSDemos/Demos/Lookup/Templates/description.md

> You can use the following properties to specify custom templates for these Lookup elements:

Reviewer: ```suggestion
You can use the following properties to specify custom templates for Lookup elements:
```

<table class="dx-table">
    <tr>
        <th>Lookup Element</th>
        <th>Property</th>
    </tr>
    <tr>
        <td>Input field</td>
        <td><a href="https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#fieldTemplate">fieldTemplate</a></td>
    </tr>
    <tr>
        <td>Drop-down list items</td>
        <td><a href="https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#itemTemplate">itemTemplate</a></td>
    </tr> 
    <tr>
        <td>Group captions</td>
        <td><a href="https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#groupTemplate">groupTemplate</a></td>
    </tr> 
</table>

> To customize elements, you can also define specific fields in data objects. Visit the [items](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/items/) documentation for a full list of such fields. Refer to the [Default Templates](/Documentation/Guide/UI_Components/Common/Templates/#Default_Templates) help topic for more information about this feature.

Reviewer: ```suggestion
To customize elements, you can also define specific fields in data objects. Visit the [items](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/items/) document for a full list of such fields. Refer to the [Default Templates](/Documentation/Guide/UI_Components/Common/Templates/#Default_Templates) help topic for more information about this feature.
```

> The Lookup contains properties to specify custom templates for the following elements:

Reviewer 2: I'm not sure that 'contains properties' is a correct term. How about the following: 
The Lookup allows you to use the following properties to specify custom templates for the lookup elements:

Author: "You can use the following properties to specify custom templates for these Lookup elements:" - used this to avoid repeating "to".

* Input field: [fieldTemplate](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#fieldTemplate)

> * Items: [itemTemplate](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#fieldTemplate)

Reviewer 2: ```suggestion
* Drop-down list items: [itemTemplate](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#fieldTemplate)
```

> * Group captions: [groupTemplate](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#groupTemplate)

Reviewer 2: Please note that React uses different properties. Seems like you need to create a separate description for the React approach.

> * Group captions: [groupTemplate](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#groupTemplate)

Reviewer 3: I think a table would be more suitable than a list in this case.

> To customize items, you can also define [specific fields](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/items/) in the data objects. In addition, the Lookup supports [3rd-party template engines](/Documentation/Guide/UI_Components/Common/Templates/#3rd-Party_Template_Engines).

Reviewer 2: >To customize items, you can also define [specific fields](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/items/) in the data objects.

Did I get it right and you suggest users to change data in the lookup's data source?

>In addition, the Lookup supports [3rd-party template engines](/Documentation/Guide/UI_Components/Common/Templates/#3rd-Party_Template_Engines).

As well as any other UI component, right?

Reviewer 3: > [specific fields](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/items/)

The URL leads to a page titled "items", which is confusing because you didn't give the context.

> In addition, the Lookup supports [3rd-party template engines](/Documentation/Guide/UI_Components/Common/Templates/#3rd-Party_Template_Engines).

This feature works only with jQuery, and even jQuery users seldom use it. Let's remove this sentence.

```suggestion
To customize items, you can also define specific fields in data objects. Visit the [items](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/items/) help topic for a full list of such fields. Refer to the [Default Templates](/Documentation/Guide/UI_Components/Common/Templates/#Default_Templates) help topic for more information about these feature.
```
